website/docs/markdown-features.mdx

```diff
@@ -195,26 +195,42 @@ Code blocks are text blocks wrapped around by strings of 3 backticks. You may ch
 
 <!-- TODO: We need to allow users to pick syntax highlighting themes (maybe other than swizzling) -->
 
-Use the matching language meta string for your code block, and Docusaurus will pick up syntax highlighting automatically, powered by [Prism React Renderer](https://github.com/FormidableLabs/prism-react-renderer). Just in case you need it, [here](https://prismjs.com/#supported-languages) is a list of supported languages and their corresponding meta strings.
+Use the matching language meta string for your code block, and Docusaurus will pick up syntax highlighting automatically, powered by [Prism React Renderer](https://github.com/FormidableLabs/prism-react-renderer).
 
 ```jsx
 console.log('Every repo must come with a mascot.');
 ```
 
-You can change the [theme](https://github.com/FormidableLabs/prism-react-renderer#theming) by simply passing `prismTheme` as `themeConfig` in your docusaurus.config.js.
+By default, the Prism [syntax highlighting theme](https://github.com/FormidableLabs/prism-react-renderer#theming)
+we use is [Palenight](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/themes/palenight.js).
+You can change this to another theme by passing `prismTheme` as `themeConfig` in your docusaurus.config.js.
 
-Example:
+For example, if you prefer to use the `dracula` highlighting theme:
 
 ```js
 // docusaurus.config.js
 module.exports = {
   themeConfig: {
-    prismTheme: require('prism-react-renderer/themes/palenight'),
+    prismTheme: require('prism-react-renderer/themes/dracula'),
   },
 };
 ```
 
-By default, the Prism theme we use is [Palenight](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/themes/palenight.js).
+By default, Docusaurus comes with this subset of
+[commonly used languages](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js).
+
+If you want to add highlighting for any of the other
+[Prism supported languages](https://prismjs.com/#supported-languages), swizzle the `CodeBlock`
+component and include the language there yourself.
+
+For example, if you want to add highlighting for the `powershell` language:
+
+```js
+// src/theme/CodeBlock/index.js
+import Prism from 'prism-react-renderer/prism';
```

Comment:
Did you ever try

```js
    <Highlight
      {...defaultProps}
      theme={prismTheme || defaultTheme}
      code={children.trim()}
      Prism={Prism}
```

instead of doing (typeof global !== 'undefined' ? global : window).Prism = Prism;

Reply:
I guess it doesnt work. Nvm